### PR TITLE
feat(console): remix guide

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add Remix app creation guide.

## Testing

Local preview.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/assets/docs/guides/web-remix/README.mdx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  },
+  "sign-in-callback": {
+    path: "/api/logto/callback",
+    redirectBackTo: "/",
+  },
+  "sign-out": {
+    path: "/api/logto/sign-out",
+    redirectBackTo: "/",
+  },
+});
```
Comment (simeng-li) [inline packages/console/src/assets/docs/guides/web-remix/README.mdx:126]: just confirm where should the developer to set their  redirect URI entered above? did not find a reference here.

File: packages/console/src/assets/docs/guides/web-remix/README.mdx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  readonly context: LogtoContext;
+};
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const context = await logto.getContext({ getAccessToken: false })(
+    request
+  );
+
+  if (!context.isAuthenticated) {
+    return redirect("/api/logto/sign-in");
```
Comment (simeng-li) [inline packages/console/src/assets/docs/guides/web-remix/README.mdx:160]: same here should we use the redirect uri path ?

File: packages/console/src/assets/docs/guides/web-remix/README.mdx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+const Home = () => {
+  const data = useLoaderData<LoaderResponse>();
+
+  return <div>Protected Route.</div>;
+};
+```
+
+</Step>
+
+<Step
```
Comment (simeng-li) [inline packages/console/src/assets/docs/guides/web-remix/README.mdx:175]: remove further readings

File: packages/console/src/assets/docs/guides/web-remix/index.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { ApplicationType } from '@logto/schemas';
+
+import { type GuideMetadata } from '../types';
+
+const metadata: Readonly<GuideMetadata> = Object.freeze({
+  name: 'Remix',
+  description:
+    'Remix is a full stack web framework that lets you focus on the user interface and work back through web standards to deliver a fast, slick, and resilient user experience.',
+  target: ApplicationType.Traditional,
```
Comment (charIeszhao) [inline packages/console/src/assets/docs/guides/web-remix/index.ts:9]: Is Remix a server side rendering lib?

**Review by charIeszhao — APPROVED** [review]
Approve to unblock. But please take a look at Simeng's comments

File: packages/console/src/assets/docs/guides/web-remix/index.ts (hunk 4 above)
Comment (wangsijie) [reply packages/console/src/assets/docs/guides/web-remix/index.ts:9]: YES

File: packages/console/src/assets/docs/guides/web-remix/README.mdx (hunk 3 above)
Comment (wangsijie) [reply packages/console/src/assets/docs/guides/web-remix/README.mdx]: updated

File: packages/console/src/assets/docs/guides/web-remix/README.mdx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/assets/docs/guides/web-remix/README.mdx]: Comment added

File: packages/console/src/assets/docs/guides/web-remix/README.mdx (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/assets/docs/guides/web-remix/README.mdx:160]: No, this is expected, redirect to the Remix app to trigger sign in.

**Pull request merged**